defaults/main.yml

```diff
@@ -159,4 +161,16 @@ ldaptoolbox_openldap_overlay_refint_olcRefintNothing: "cn=nothing,{{ ldaptoolbox
 # Dynamic groups (dynlist)
 ldaptoolbox_openldap_overlay_dynlist_olcDlAttrSet: "groupOfURLs memberURL member+memberOf@groupOfNames*"
 
+#############
+# Data import
+#############
+
+ldaptoolbox_openldap_data_prefix: 'openldap-data'
+# if not defined the default population template will be used which is fine for test purposes
```

Comment:
I have kept the import task, but IMO, it is not really a task which should be automatized

Reply:
I have also added custom data to illustrate the behaviour